hack/deploy-scheduler-estimator.sh

```diff
@@ -4,6 +4,12 @@ set -o errexit
 set -o nounset
 
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+
+#define karmada control plane image registry and version
+KARMADA_REGISTRY=${KARMADA_REGISTRY:-"swr.ap-southeast-1.myhuaweicloud.com/karmada"}
+KARMADA_VERSION=${KARMADA_VERSION:-"latest"}
+KUBE_REGISTRY=${KUBE_REGISTRY:-"k8s.gcr.io"}
```

Comment:
We seemingly don't use this variate `KUBE_REGISTRY` in this script.

Reply:
GOT IT